src/components/molecules/SfProperty/SfProperty.html

```diff
@@ -0,0 +1,6 @@
+<div class="sf-property">
+	<slot>
+        <span class="sf-property__property">{{ property }}:</span>
```

Comment:
Hey @aniamusial , wdyt about `sf-property__name` here? Then you don't repeat _property_...

Reply:
or `sf-property__title`

Reply:
Sure we can change that, I was just follwing the proposed API :)